app/presenters/queries/content_item_presenter.rb

```diff
@@ -54,37 +55,46 @@ def total
     private
 
       def results
-        @results ||= execute_query(ordered_fields)
+        @results ||= execute_query(query)
       end
 
-      def ordered_fields
-        select_fields(order_and_paginate)
-      end
-
-      def full_scope
-        search(join_supporting_objects(latest))
-      end
-
-      def latest
-        ::Queries::GetLatest.call(self.scope.joins(:document))
+      def query
```

Comment:
i agree with the suggestion. looking through, i think the code is fine (given the goal at this point isn't "fix everything")